src/fonduer/supervision/labeler.py

```diff
@@ -18,12 +18,27 @@
 
 
 def load_gold_labels(session, cand_lists, annotator_name="gold"):
-    """Load the sparse matrix for the specified annotator."""
+    """Load the sparse matrix for the specified annotator.
+
+    :param session: The database session.
+    :param cand_lists: The candidates to get gold labels for.
+    :type cand_lists: List of list of candidates.
```

Comment:
-`candidate objects` or `Candidate`s to signal it refers to the class
-Why is it a list of lists? Related to multitask?

Reply:
>  -Why is it a list of lists? Related to multitask?

Yes, a list of lists of candidates for each relation. E.g. [[PartTemp Cands], [PartVolt Cands], etc.]